server/templates/staff/course/enrollment/enrollment.unenrolled.html

```diff
@@ -0,0 +1,71 @@
+{% extends "staff/base.html" %}
+{% import 'staff/_formhelpers.html' as forms %}
+
+{% block title %} Un-Enrolled Submitters - {{ current_course.display_name_with_semester }}{% endblock %}
+
+{% block main %}
+<section class="content-header">
+    <h1>
+        {{ current_course.display_name_with_semester }} Un-Enrolled Submitters
+        <small>{{ current_course.offering }}</small>
+    </h1>
+    <ol class="breadcrumb">
+        <li><a href="{{ url_for(".course", cid=current_course.id) }}">
+            <i class="fa fa-university"></i> {{ current_course.offering }}
+        </a></li>
+        <li class="active"><a href="{{ url_for('.unenrolled', cid=current_course.id) }}">
+            <i class="fa fa-user-o"></i> Un-Enrolled Submitters</a>
+        </li>
+    </ol>
+</section>
+<section class="content">
+    {% include 'alerts.html' %}
+    <div class="row">
+        <div class="col-md-9 col-xs-12">
+            <div class="box">
+                <div class="box-header">
+                    <h3 class="box-title"><span> {{ title }} </span></h3>
+                    <div class="box-tools">
+                        <div class="box-tools pull-right">
+                            <button type="button" class="btn btn-box-tool" data-widget="collapse" data-toggle="tooltip" title="Collapse">
+                                <i class="fa fa-minus"></i>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+
+                <!-- /.box-header -->
+                <div class="box-body table-responsive no-padding table-loading" id="{{ title }}-list">
+                    <table class="table table-hover">
+                        <thead>
+                            <th class="sort" data-sort="email">User</th>
+                            <th class="sort" data-sort="sid">SID</th>
+                        </thead>
+
+                        <tbody class="list">
+                            {% for unenrolled_submitter in unenrolled_submitters %}
+                            <tr>
+                                <td class="email">
+                                    <a href="{{ url_for('.student_view', cid=current_course.id, email=unenrolled_submitter.email) }}">
+                                        {{ unenrolled_submitter.email }}
+                                    </a>
+                                </td>
+                                <td class="sid">{{ unenrolled_submitter.id }}</td>
```

Comment:
The ID isn't particularly useful (it's our database ID)  Maybe their name could go here instead?